services/blocks/standard/service.go

```diff
@@ -152,10 +153,16 @@ func New(ctx context.Context, params ...Parameter) (*Service, error) {
 		consolidationRequestsSetter: consolidationRequestsSetter,
 		chainTime:                   parameters.chainTime,
 		refetch:                     parameters.refetch,
+		blobEnable:                  parameters.blobEnable,
 		activitySem:                 parameters.activitySem,
 		syncCommittees:              make(map[uint64]*chaindb.SyncCommittee),
 	}
 
+	log.Info().
+		Bool("blobEnable", parameters.blobEnable).
+		Bool("refetch", parameters.refetch).
+		Int64("startSlot", parameters.startSlot).
+		Msg("Blocks Service initialized")
```

Comment:
I don't think we need this log line, but if you feel it's very useful then please set it to `Trace` level.